charts/bindplane/templates/bindplane-jobs.yaml

```diff
@@ -217,13 +187,13 @@ spec:
               value: {{ .Values.backend.postgres.sslmode }}
             {{- if .Values.backend.postgres.sslsecret.sslrootcertSubPath }}
             - name: BINDPLANE_POSTGRES_SSL_ROOT_CERT
-              value: /postgres-tls/ca.crt
+              value: /postgres-tls/{{ .Values.backend.postgres.sslsecret.sslrootcertSubPath }}
             {{- end }}
             {{- if .Values.backend.postgres.sslsecret.sslcertSubPath }}
             - name: BINDPLANE_POSTGRES_SSL_CERT
-              value: /postgres-tls/client.crt
+              value: /postgres-tls/{{ .Values.backend.postgres.sslsecret.sslcertSubPath }}
             - name: BINDPLANE_POSTGRES_SSL_KEY
-              value: /postgres-tls/client.key
+              value: /postgres-tls/{{ .Values.backend.postgres.sslsecret.sslkeySubPath }}
```

Comment:
We need to set the file names like this, in order to match the changes in values.yaml. The default behavior is the same, the default file names are `ca.crt`, `client.crt`, `client.key`.

This change combined w/ the change in values.yaml allows the user to define the certificate file names instead of catering to whatever name we pick. This is important because some users may be using non defaults for the sub path options.

This was not a concern before, because the `initContainer` would mount the files using the user defined subpaths but overriding the names to `ca.crt`, `client.crt`, `client.key`.